### PR TITLE
2298 spanish translations

### PR DESCRIPTION
**Pull request by calvinfroedge**
Updated language translations using both automatic and manual checks

Resolves https://github.com/bitshares/bitshares-ui/issues/2298

### CODE REVIEW

**Comment by calvinfroedge** [comment]
WTF! There must be some process that overwrote this...

**Comment by calvinfroedge** [comment]
@sschiessl-bcp I just placed the Spanish file manually...

**Comment by calvinfroedge** [comment]
@sschiessl-bcp Please check again

**Comment by calvinfroedge** [comment]
@sschiessl-bcp bump

File: app/assets/locales/locale-es.json [hunk 1]
```diff
@@ -42,8 +42,8 @@
         "hide_hidden": "Active",
         "hide_ignored": "Ocultar cuentas ocultas",
         "history": "Historial",
-        "history_error": "Fallo en la descarga del historial de cuenta, por favor inténtelo otra vez",
-        "home": "Casa",
+        "history_error": "Error al descargar el historial de la cuenta, inténtalo de nuevo",
```
[the remark this answers is not in the document]
Comment (calvinfroedge) [reply app/assets/locales/locale-es.json]: Nobody would ever use "Casa" for "Home Page"

[the remark this answers is not in the document]
Comment (sschiessl-bcp) [reply app/assets/locales/locale-es.json]: My comment was to the `history_error`, I think `account.home` is currently ununsed :)

[the remark this answers is not in the document]
Comment (calvinfroedge) [reply app/assets/locales/locale-es.json]: Well the old translation is just not a very good translation...for one "Error" is more clear than "Fallo", and "de nuevo" means "again" whereas "otra vez" means another time.

[the remark this answers is not in the document]
Comment (calvinfroedge) [reply app/assets/locales/locale-es.json]: I made quite a few changes in addition to converting all the strings that were in English...

**Pull request merged**